src/main/java/io/r2dbc/postgresql/PostgresqlConnectionConfiguration.java

```diff
@@ -186,7 +261,8 @@ public Builder applicationName(String applicationName) {
          */
         public PostgresqlConnectionConfiguration build() {
             return new PostgresqlConnectionConfiguration(this.applicationName, this.connectTimeout, this.database, this.forceBinary, this.host, this.options, this.password, this.port, this.schema,
-                this.username);
+                this.username,
```

Comment:
It would make sense to check whether the configured SSL key/cert/root cert actually exist to validate and fail early if the files are missing.